flang/test/Lower/Intrinsics/dsecnds.f90

```diff
@@ -0,0 +1,23 @@
+! RUN: bbc -emit-fir -hlfir=false %s -o - | FileCheck %s
```

Comment:
Please could you test using HLFIR. The old lowering directly to FIR is deprecated and unused.

```suggestion
! RUN: bbc -emit-hlfir %s -o - | FileCheck %s
```